bitnami/cassandra/templates/statefulset.yaml

```diff
@@ -269,35 +297,39 @@ spec:
               containerPort: {{ .Values.containerPorts.intra | default "7000" }}
               {{- if .Values.hostNetwork }}
               hostPort: {{ .Values.containerPorts.intra }}
-              {{- end }}
+            {{- end }}
             - name: tls
               containerPort: {{ .Values.containerPorts.tls | default "7001" }}
               {{- if .Values.hostNetwork }}
               hostPort: {{ .Values.containerPorts.tls }}
-              {{- end }}
+            {{- end }}
             - name: jmx
               containerPort: {{ .Values.containerPorts.jmx | default "7199" }}
               {{- if .Values.hostNetwork }}
               hostPort: {{ .Values.containerPorts.jmx }}
-              {{- end }}
+            {{- end }}
             - name: cql
               containerPort: {{ .Values.containerPorts.cql | default "9042" }}
               {{- if .Values.hostNetwork }}
               hostPort: {{ .Values.containerPorts.cql }}
-              {{- end }}
+            {{- end }}
             {{- if .Values.cluster.enableRPC }}
             - name: thrift
               containerPort: {{ .Values.containerPorts.thrift | default "9160" }}
               {{- if .Values.hostNetwork }}
               hostPort: {{ .Values.containerPorts.thrift }}
-              {{- end }}
             {{- end }}
+          {{- end }}
```

Comment:
Could you revert these changes? they are only tabulations

Reply:
Done